data/src/main/java/org/cryptomator/data/cloud/webdav/network/ConnectionHandlerHandlerImpl.kt

```diff
@@ -31,6 +33,10 @@ class ConnectionHandlerHandlerImpl @Inject internal constructor(httpClient: WebD
 		webDavClient.writeFile(url, inputStream)
 	}
 
+	fun writeFile(url: String, inputStream: InputStream, modifiedDate: Date) {
+		webDavClient.writeFile(url, inputStream, modifiedDate)
+	}
+
```

Comment:
`fun writeFile(url: String, inputStream: InputStream)` seems to be unused now, can we just add this `modifiedDate` property to the existing function instead of creating a new one or do I miss something?